model/map/src/main/java/org/keycloak/models/map/singleUseObject/MapSingleUseObjectProvider.java

```diff
@@ -42,7 +40,7 @@
 /**
  * @author <a href="mailto:mkanis@redhat.com">Martin Kanis</a>
  */
-public class MapSingleUseObjectProvider implements ActionTokenStoreProvider, SingleUseObjectProvider {
+public class MapSingleUseObjectProvider implements SingleUseObjectProvider {
```

Comment:
This file still mentions ActionTokens. I will add some ideas that may be added if you like it. 

- It seems we can now remove `singleUseEntityToAdapter` method
- `actionTokenStoreTx` should be renamed to something like `singleUseObjectTx`
- The usage of `ActionTokenValueModel` is a little bit weird now. Should we rename it to `SingleUseObjectModel`? 
- Also, the presence of `ObjectKey` there is weird. For example, in HotRod it is not necessary at all, so it sounds like an implementation detail for JPA so maybe it should be present only there? WDYT? Should we create a new issue for that?

Reply:
Thanks. I addressed these suggestions. I renamed both `ActionTokenValueModel` and `ActionTokenKeyModel` and some related classes and variables. 
I agree regarding `ObjectKey` that it is weird and should handle it in a separate issue.

Reply:
I created an issue: [#13989](https://github.com/keycloak/keycloak/issues/13989)